### DISCUSSION TITLE
Take picture without QR code

### DISCUSSION THREAD

**Discussion by peterwup** [photobooth-app/photobooth-app#88] [Q&A]
I understand that sharing "only" works with internet access, not with a local accespoint. 

Is it possible to hide the QR code for such cases ? 
Shareservice Enabled = FALSE
Shareservice Url = ""
But QR Code is still shown.

**Comment by mgineer85** [comment]
Hi, 
thank you for reporting. This is a bug, I will fix and update a new release very soon. I keep you updated.
https://github.com/mgrl/photobooth-app/issues/89

**Comment by mgineer85** [comment]
Added option to hide QR code in gallery released: https://pypi.org/project/photobooth-app/
Please mark this as answer if solved for you.
